.github/workflows/ci.yml

```diff
@@ -923,3 +923,43 @@ jobs:
     - name: run ./ci/docfx.cmd
       shell: cmd
       run: ./ci/docfx.cmd
+
+  w3c_trace_context_compliance_v1:
+    name: W3C Distributed Tracing Validation V1
+    runs-on: ubuntu-24.04
```

Comment:
nit - ubuntu-latest should be fine, unless we have some dependency on particular distro version.

Reply:
No, there is no dependency on this distro version, I can change it to ubuntu-latest. I don't have a strong reason for choosing specific version.

Reply:
I did not realize that [ubuntu-latest is actually 22.04](https://github.com/actions/runner-images), I thought it would be 24.04. 

So it has gcc-10 instead of gcc-14.